docs/workflow/debugging/libraries/windows-instructions.md

```diff
@@ -52,7 +43,7 @@ The following commands will properly configure the debugging extension and fix s
 .symfix
 .srcfix
 .reload
-!load C:\corefx\packages\runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR\<version>\tools\sos
+!load C:\runitme\coreclr\runtime.win7-x64.Microsoft.NETCore.Runtime.CoreCLR\<version>\tools\sos
```

Comment:
Typo

Reply:
Actually SOS isn't part of the runtime anymore.  If you have installed an recent internal version of the windows debugger (windbg), SOS will be loaded automatically from the extension gallery.  If it doesn't install automatically when coreclr.dll is loaded, see this [doc](https://github.com/dotnet/diagnostics/blob/master/documentation/installing-sos-windows-instructions.md#installing-sos-on-windows).

Reply:
Let's update the doc regarding sos in a different PR.

Reply:
Those directions are wrong now for .NET 3.x and above.  SOS is shipped out of band, not part of the runtime anymore but is automatically loaded from the extension gallery for recent internal windbg versions.